components/automate-cli/cmd/chef-automate/config.go

```diff
@@ -54,6 +54,7 @@ const (
 	patching                        = "Config Patch"
 	setting                         = "Config Set"
 	waitTimeout                     = "wait-timeout"
+	automateHaPath                  = "/hab/var/automate-ha/"
```

Comment:
@iamazzeez  Does the slash needed at the end?

Reply:
Thats optional.